### ISSUE TITLE
Fileinfo replacement

### ISSUE THREAD

**Issue by alexcos20** [oceanprotocol/market#348]
Market should use ocean.provider.fileinfo  (https://github.com/oceanprotocol/ocean.js/blob/main/src/provider/Provider.ts#L81) instead of using it's own endpoint
(https://github.com/oceanprotocol/market/blob/main/src/utils/index.ts#L45)

**Comment by mihaisc** [comment]
Once this is done we should remove this endpoint `https://fileinfo.oceanprotocol.com`

**Comment by kremalicious** [comment]
duplicate of #318
